src/nanovg_mtl.m

```diff
@@ -1558,3 +1558,16 @@ void mnvgReadPixels(NVGcontext* ctx, int image, int x, int y, int width,
          fromRegion:MTLRegionMake2D(x, y, width, height)
         mipmapLevel:0];
 }
+
+int mnvgCreateImageFromHandle(NVGcontext* ctx, void* textureId, int imageFlags) {
```

Comment:
Please move the implementation to the position between `mnvgCommandQueue()` and `mnvgDevice()`.

Reply:
done